packages/yoroi-extension/app/components/wallet/add-wallet-revamp/AddWalletCard.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Node } from 'react';
+import { Box, Typography } from '@mui/material';
+import styles from './AddWalletCard.scss';
+
+type Props = {|
+  label: string,
+  imageSrc: string,
+  onClick(): void,
+|};
+
+export default function AddWalletCard(props: Props): Node {
```

Comment:
Add wallet card seems not to fit into a generic component. We use this type of card on other funnels, eg. Restore. I think we should change the name and structure to fit this more generic purpose.